shell/src/main/java/alluxio/shell/command/LsCommand.java

```diff
@@ -41,6 +42,41 @@
   public static final String STATE_FILE_IN_MEMORY = "In Memory";
   public static final String STATE_FILE_NOT_IN_MEMORY = "Not In Memory";
 
+  private static final Option FORCE_OPTION =
+      Option.builder("f")
+          .required(false)
+          .hasArg(false)
+          .desc("force")
+          .build();
+
+  private static final Option LIST_DIR_AS_FILE_OPTION =
+      Option.builder("d")
+          .required(false)
+          .hasArg(false)
+          .desc("list directories as plain files")
+          .build();
+
+  private static final Option LIST_HUMAN_READABLE_OPTION =
+      Option.builder("h")
+          .required(false)
+          .hasArg(false)
+          .desc("print human-readable format sizes")
+          .build();
+
+  private static final Option LIST_PINNED_FILES_OPTION =
+      Option.builder("p")
+          .required(false)
+          .hasArg(false)
+          .desc("list all pinned files")
+          .build();
+
+  private static final Option RECURSIVE_OPTION =
+      Option.builder("R")
+          .required(false)
+          .hasArg(false)
+          .desc("recursive")
```

Comment:
Since we are now in the command, we could expand the description a bit to be more specific. E.g., for this one, we can change it to `.desc("list subdirectories recursively")`